### PR TITLE
Adding three new features

### PR DESCRIPTION
**Pull request by techfreaque**
pull request as discussed with guillaum, let me know if you have any questions

### CODE REVIEW

**Comment by GuillaumeDSM** [comment]
this PR replaces https://github.com/Drakkar-Software/OctoBot/pull/1992, right ?

File: octobot/strategy_optimizer/strategy_design_optimizer.py [hunk 1]
```diff
@@ -694,6 +697,12 @@ async def _generate_and_store_backtesting_runs_schedule(self):
         await self._save_run_schedule(runs)
         return runs
 
+    def _shuffle_and_select_runs(self, runs, select_size=None) -> dict:
```
Comment (Herklos) [inline octobot/strategy_optimizer/strategy_design_optimizer.py:700]: 👍

**Review by Herklos — APPROVED** [review]
👍

**Comment by techfreaque** [comment]
> 

Yes this replaces https://github.com/Drakkar-Software/OctoBot/pull/1992
I just closed it

**Pull request merged**